src/store/modules/program/state.js

```diff
@@ -0,0 +1,9 @@
+import { INITIAL_ACTION_STATE } from "../../../helper/constants";
+
+export default {
+  list: [],
+  actions: {
```

Comment:
to be discussed: why do we use the action entry in the state ? Does that make sense?